README.md

```diff
@@ -194,17 +194,19 @@ specified target's database and downloads it to `tmp/`.
 `geordi dump staging -l` (with a Capistrano deploy target and the `--load`
 option) sources the dump into the development database after downloading it.
 
-If you are using multiple databases per environment, Geordi defaults to the
-"primary" database, or the first entry in database.yml. To dump a specific
-database, pass the database name like this:
+If you are using multiple databases per environment, you have to specify the
+database name. To dump a specific database, pass the database name with the
+`-d`/`--database` option. To load a dump into a specific database, pass the
+database name with the `-L`/`--load-database` option.
 
-    geordi dump -d primary
-
-Loading a dump into one of multiple local databases is not supported yet.
+For dumping, geordi defaults to the "primary" database, or the first entry in
+ database.yml. For dumping, these defaults are deactivated to avoid unwanted
```

Comment:
` database.yml` leading space might be removed

Reply:
Also, please revert the changes to this paragraph and instead apply this:
```suggestion
When used with the `load` option ("dump and source"), the `database`
option will be respected both for the remote *and* the local database.
If these should not match, please issue separate commands for dumping
(`dump -d`) and sourcing (`dump -l -d`).

Reply:
(Of course these changes need to go to lib/geordi/commands/dump.rb)